docs/adr/0016-some-case-list-email-present-wrong.md

```diff
@@ -0,0 +1,21 @@
+# 16. Fix some wrong cases email list presents
+
+Date: 2022-09-13
+
+## Status
+
+- Issue: Mailbox A has cache with more than 20 filtedted emails.
+Then comeback this mailbox in the next time, mails is not show correctly, read-emails can not show
```

Comment:
`Then come back this mailbox in the next time, read-emails are not showed correctly`